Doc/library/plistlib.rst

```diff
@@ -76,16 +74,12 @@ This module defines the following functions:
    The parser for the binary format raises :exc:`InvalidFileException`
    when the file cannot be parsed.
 
-   .. versionadded:: 3.4
```

Comment:
why remove this `versionadded`? it's a new api from 3.4 in https://bugs.python.org/file32754/issue-14455-v10.txt. the downstairs `versionadded` have this problem too.

Reply:
The `versionadded` tags relate to the deprecation of the old plistlib API after the new one superseded it. Since the old API has been removed (#15615) I thought it would be a good idea to clean up the documentation. Also, Python versions older than 3.5 are not supported, so I figured the tags didn't mean much any way.

I'm happy to re-add all the `versionadded` tags if desired?

Apologies if I'm not following the standard conventions for making changes to the documentation!

Reply:
I agree, the versionadded directives for the various functions should not be removed.